docs/source/dataset_script.mdx

```diff
@@ -270,85 +270,27 @@ def _generate_examples(self, filepath):
 ## Testing data and checksum metadata
```

Comment:
I would suggest renaming this section to `Generate dataset metadata`:
- Sections at the `##` level start with a verb
- The metadata contains more than just checksum 
```suggestion
## Generate dataset metadata
```